.github/actions/javascript/postTestBuildComment/action.yml

```diff
@@ -3,22 +3,22 @@ description: "Mark pull requests as deployed on production or staging"
 inputs:
     PR_NUMBER:
         description: "Pull request number"
-        required: true
+        required: false
     GITHUB_TOKEN:
         description: "Github token for authentication"
         default: "${{ github.token }}"
     ANDROID:
         description: "Android job result ('success', 'failure', 'cancelled', or 'skipped')"
-        required: true
+        required: false
     DESKTOP:
         description: "Desktop job result ('success', 'failure', 'cancelled', or 'skipped')"
-        required: true
+        required: false
     IOS:
         description: "iOS job result ('success', 'failure', 'cancelled', or 'skipped')"
-        required: true
+        required: false
     WEB:
         description: "Web job result ('success', 'failure', 'cancelled', or 'skipped')"
-        required: true
+        required: false
```

Comment:
Won't this change will affect our existing builds?

Reply:
I don't think so. This action is only used in `testBuild.yml` where all four platforms are passed. I modified the script so it checks if given platform is included so there shouldn't be any problems

Reply:
Okay thanks. Once last question: If we are now checking for the script elsewhere, why was this change necessary? It seems to me that requiring at the action level is a bit safer than a check deeper within a script.

Reply:
`action.yml` is [file with metadata](https://docs.github.com/en/actions/sharing-automations/creating-actions/about-custom-actions#types-of-actions) and specifies input. I modified the script so not all platforms are required. If input for given platform is not specified there will be N/A in table with statuses and links. 

It's probably not ideal but I wanted to reuse already written action

Reply:
Ah okay. So We are reusing action logic, but need to account for platforms not being provided. It sounds reasonable.